src/composables/ViewRoutingSlip/usePaymentInformation.ts

```diff
@@ -51,6 +70,20 @@ export default function usePaymentInformation (_, context) {
     return linkedRoutingSlips.value && linkedRoutingSlips.value.children.length > 0 && linkedRoutingSlips.value.children[0].totalUsd && linkedRoutingSlips.value.children[0].totalUsd > 0
   })
 
+  function adjustRoutingSlipHandler () {
+    try {
+      adjustRoutingSlip()
+      isEditable.value = !isEditable.value
+    } catch (error: any) {
+      // eslint-disable-next-line no-console
+      console.error('error ', error?.response)
```

Comment:
Error just gets swallowed here?

Reply:
Yes error gets swallowed, but UI doesn't reset if it has error occurs.